src/avram/criteria_extensions/extract_criteria.cr

```diff
@@ -0,0 +1,132 @@
+module Avram
+  enum ChronoUnits
+    Century
+    Day
+    Decade
+    Dow
+    Doy
+    Epoch
+    Hour
+    Isodow
+    Isoyear
+    Julian
+    Microseconds
+    Millennium
+    Milliseconds
+    Minute
+    Month
+    Quarter
+    Second
+    Timezone
+    Timezone_hour
+    Timezone_minute
```

Comment:
Curious about these. Is this because of the Symbol arg override? I'm assuming doing TimezoneHour then passing :timezone_hour fails?

Reply:
Actually, it was a complete mishap from my part, dunno why but I was convinced that they were with no underscore :/ either way, I changed them to be correct now :) which also made me learn about camelcase and underscore methods XD